src/handler.ts

```diff
@@ -16,7 +15,7 @@ export const handle: Handler = async () => {
     ])
     return { success: true }
   } catch (e) {
-    error(e)
+    console.error(e)
```

Comment:
I don't see a strong case for the value provided by [the package](https://github.com/therockstorm/utils/blob/master/src/index.ts) so I'm onboard for a `console.error` regardless of env